### PR TITLE
Add/plant to place

### PR DESCRIPTION
**Pull request by frank12421**

### CODE REVIEW

File: pages/lists/addplanttoplace.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
+import { plants } from "@/db/db";
+import { StyledButton, StyledLink } from "@/components/StyledButton";
+import {
+  StyledCard,
+  StyledCardUl,
+  StyledContentContainer,
+  StyledLi,
```
Comment (CJAkkuec) [inline pages/lists/addplanttoplace.js:7]:
```suggestion
import {
  StyledCard,
  StyledCardUl,
  StyledContentContainer,
  StyledLi,
```

In general I'd suggest `StyledCardList` for `StyledCardUl` and `StyledCardListItem` for `StyledLi`.

File: pages/lists/addplanttoplace.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+  return (
+    <main>
+      <h1>Pflanze auswählen - Grow Green</h1>
+      <StyledLink color="darkgreen" href="/">
+        zurück
+      </StyledLink>
+      <>
+        {plantsSorted.map((plant) => {
+          return (
+            <section key={plant.id}>
```
Comment (CJAkkuec) [inline pages/lists/addplanttoplace.js:39]:
```suggestion
            <section key={plant.id}>
```
I don't think you need a `section` for every card. 🤔

**Review by CJAkkuec — CHANGES_REQUESTED** [review]
Just a couple of small notes 👍

**Comment by frank12421** [comment]
Many thanks for the notes. All done. 
Thanks for reviewing again.

File: pages/lists/addplanttoplace.js [hunk 3]
```diff
@@ -36,25 +39,31 @@ export default function AddPlantToPlaceList({ places, setPlaces }) {
       <>
         {plantsSorted.map((plant) => {
           return (
-            <section key={plant.id}>
+            <React.Fragment key={plant.id}>
```
Comment (djfarly) [inline pages/lists/addplanttoplace.js:42]: This works but is not really needed. You can just add the `key` to `StyledCard`:

```jsx
<StyledCard border={"green"} key={plant.id}>
```

then you can get rid of the Fragment alltogether.

File: pages/lists/placelist.js [hunk 4]
```diff
@@ -22,7 +22,7 @@ export default function PlaceList({ places }) {
               : `/pictures/Place-People.png`;
 
           return (
-            <section key={place.id}>
+            <React.Fragment key={place.id}>
```
Comment (djfarly) [inline pages/lists/placelist.js:25]: same here ;)

File: pages/lists/plantlist.js [hunk 5]
```diff
@@ -18,12 +19,12 @@ export default function PlantList() {
...
         {plants.map((plant) => {
           const imgSrc =
-            plant.typ === "Normal"
+            plant.type === "Normal"
               ? "/pictures/Plant1.png"
               : "/pictures/Plant2.png";
 
           return (
-            <section key={plant.id}>
+            <React.Fragment key={plant.id}>
```
Comment (djfarly) [inline pages/lists/plantlist.js:27]: and here

**Review by djfarly — APPROVED** [review]
just a tiny note - otherwise LGTM :)

You don't need to submit for re-re-review if you only remove the Fragments. 😄

File: pages/lists/addplanttoplace.js (hunk 3 above)
Comment (frank12421) [reply pages/lists/addplanttoplace.js:42]: Oh, I tried, but kept getting iene errors in Firefox. Therefore the way with the fragment. I try it again.

**Pull request merged**